server/database.go

```diff
@@ -153,6 +153,11 @@ func (s *Server) registerDatabaseRoutes(g *echo.Group) {
 		if database == nil {
 			return echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Database not found with ID %d", id))
 		}
+		// Wildcard(*) database is used to connect all database at instance level.
+		// Do not return it via `get database by id` API.
+		if database.Name == api.AllDatabaseName {
+			return echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Database not found with ID %d", id))
```

Comment:
```suggestion
			return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Database %d is a wildcard *", id))
```

Reply:
Please see the suggested editing. We may have legit use case to return wildcard db in the future, but at least when that happens, the error message will give us a clue.